relay-general/src/types/meta.rs

```diff
@@ -474,7 +474,7 @@ impl Meta {
     }
 
     fn upsert(&mut self) -> &mut MetaInner {
-        self.0.get_or_insert_with(|| Box::new(MetaInner::default()))
+        self.0.get_or_insert_with(|| Box::<MetaInner>::default())
```

Comment:
This should work without type hints.

```suggestion
        self.0.get_or_insert_with(|| Box::default())
```

Reply:
I haven't tried it, just used what clippy suggested. Will try it now and make a change.